classes/admin.php

```diff
@@ -25,23 +25,23 @@ public function add_addl_menus(){
 			$options = $simple_map->get_options();
 
 			add_submenu_page( 'edit.php?post_type=sm-location',
-				__( 'SimpleMap: General Options', 'simplemap' ),
-				__( 'General Options', 'simplemap' ),
+				__( 'SimpleMap: General Options', 'SimpleMap' ),
+				__( 'General Options', 'SimpleMap' ),
 				apply_filters( 'sm-admin-permissions-sm-options',
 					'manage_options' ), 'simplemap', array(
 					&$sm_options,
 					'print_page',
 				) );
 			add_submenu_page( 'edit.php?post_type=sm-location',
-				__( 'SimpleMap: Import / Export CSV', 'simplemap' ),
-				__( 'Import / Export CSV', 'simplemap' ), 'publish_posts',
+				__( 'SimpleMap: Import / Export CSV', 'SimpleMap' ),
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line indented incorrectly; expected at least 20 spaces, found 16
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->